problems/atbash/atbash.java

```diff
@@ -0,0 +1,29 @@
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class AtBash {
+
+    private static char[] letters = IntStream.rangeClosed('a', 'z')
+        .mapToObj(c -> "" + (char) c).collect(Collectors.joining()).toCharArray();
+
+    private static String cipher(String text) {
+        StringBuilder cipheredText = new StringBuilder();
+        int index;
+        for (char c : text.toCharArray()) {
+            index = IntStream.range(0, letters.length)
+                .filter(i -> letters[i] == Character.toLowerCase(c))
+                .findFirst()
+                .orElse(-1);
```

Comment:
You don't have to use Streams everywhere. (I did these things too)
Your index is just c - 'a'. (If you want to support uppercase letters you could set your index to Character.toLowerCase(c) - 'a')